### DISCUSSION TITLE
What are the settings for these command line options.

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#821] [Q&A] [answered]
Since most command line options can now be found in settings, could someone tell me the Settings for these?

--opt-split-attention
--disable-nan-check
--opt-sub-quad-attention

---

Also, I have an AMD RX 6600 XT 8Gb card, running in Windows via DirectML.
The following options were what I was using in lshqqytiger's fork. Are they the best options for my situation? (I get around 1.5s per iter).

--opt-split-attention --disable-nan-check --medvram --always-batch-cond-uncond --opt-sub-quad-attention --no-half-vae

**Comment by vladmandic** [comment, marked as answer]
opt-split-attention and opt-sub-quad-attention are mutually exclusive.  
the fact that you thought you had both in a1111 is just because it does no checks.
pick one in UI Settings -> Stable Diffusion -> Cross-attention optimization method

nan check is in UI Settings -> Compute Settings
